{"name": "Get account by id invalid -- @1.2 ", "status": "failed", "statusDetails": {"message": "compare.UnmetExpectation: Expected 400 to equal 404\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\project_steps.py\", line 42, in step_impl\n    expect(int(status_code)).to_equal(context.response.status_code)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 173, in to_equal\n    ensure(self.value == other, True, message)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 151, in ensure\n    raise UnmetExpectation(message)\n"}, "steps": [{"name": "Given I set up a \"GET\" request to \"/accounts/{abcd}\" endpoint", "status": "passed", "start": 1556287309476, "stop": 1556287309489}, {"name": "When I send the request", "status": "passed", "start": 1556287309489, "stop": 1556287310173}, {"name": "Then I get a \"400\" status code as response", "status": "failed", "statusDetails": {"message": "compare.UnmetExpectation: Expected 400 to equal 404\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\project_steps.py\", line 42, in step_impl\n    expect(int(status_code)).to_equal(context.response.status_code)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 173, in to_equal\n    ensure(self.value == other, True, message)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 151, in ensure\n    raise UnmetExpectation(message)\n"}, "start": 1556287310174, "stop": 1556287310176}, {"name": "And I should see a messages error: The id isn't valid", "status": "skipped", "start": 1556287310177, "stop": 1556287310177}], "parameters": [{"name": "id", "value": "abcd"}], "start": 1556287309474, "stop": 1556287310177, "uuid": "95cbe5da-e443-41a8-8d5e-a5c543cae019", "historyId": "1ef71e724e16bbe7c38fe23df6510a9a", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Get account by id"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}